lib/App/pmuninstall.pm

```diff
@@ -308,8 +308,7 @@ sub is_local_lib {
     my ($self, $file) = @_;
     return unless $self->{local_lib};
 
-    my $local_lib_base = quotemeta File::Spec->canonpath(Cwd::realpath($self->{local_lib}));
-    $file = File::Spec->canonpath($file);
+    my $local_lib_base = quotemeta Cwd::realpath($self->{local_lib});
```

Comment:
Yeah, I saw that this didn't work either. `realpath` can return a 8.3-short-name for `$file` while `$local_lib_base` uses a full name on Win32.